aidial_adapter_openai/utils/auth.py

```diff
@@ -1,7 +1,41 @@
+import time
 from typing import Mapping, Optional
 
+from azure.core.credentials import AccessToken
+from azure.identity import DefaultAzureCredential
+from fastapi import Request
+from openai import util
+from openai.util import ApiType
 from pydantic import BaseModel
 
+default_credential = DefaultAzureCredential()
+access_token: AccessToken | None = None
+
+EXPIRATION_WINDOW_IN_SEC = 10
```

Comment:
let's support env variable for the expiration window

Reply:
fixed